frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -64,13 +64,15 @@ export function Send({ accounts, code }: Props) {
     }
   };
 
-  const onRawInputChange = (event: ChangeEvent<HTMLInputElement>) => {
-    const target = event.target as HTMLInputElement;
-    setRawInput(target.value);
-  };
-
-  const onChangeActiveScanQr = (active: boolean) => {
-    setActiveScanQr(active);
+  const onInvoice = (invoice: string) => {
+    // TODO: ensure this is a valid invoide?
```

Comment:
`parseInput` calls `getParseInput` which validates the rawInput. Its will also determine if its a bolt11 invoice or lnurl-pay

Reply:
then I remove this if  on the next line, that with `setRawInputError('is not a valid invoice');` ?

Reply:
Does `ScanQRVideo` ever not return a string?

Reply:
It's always a string (coming from qr-scanner onResult callback), I remove the check.

Reply:
fixed